File: sdkwork-backend-sdk-csharp/Models/MsgMusicContent.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class MsgMusicContent
    {
        public string? Content { get; set; }
        public MusicMediaResource? Resource { get; set; }
        public string? Title { get; set; }
        public string? Artist { get; set; }
        public string? Album { get; set; }
        public int? Duration { get; set; }
    }
}

File: sdkwork-backend-sdk-go/types/msg_music_content.go
package types

// IM音乐消息内容类
type MsgMusicContent struct {
	Content string `json:"content"`
	Resource MusicMediaResource `json:"resource"`
	Title string `json:"title"`
	Artist string `json:"artist"`
	Album string `json:"album"`
	Duration int `json:"duration"`
}
